packages/wrangler/lib/functions/buildWorker.ts

```diff
@@ -0,0 +1,42 @@
+import path from "path";
```

Comment:
please rename this folder to pages/functions. lib is too generic, and we want to be clear that functions is a pages-only thing right now.

Reply:
I've moved it into `src/pages/functions`.